src/renderer/dx/DxFontRenderData.h

```diff
@@ -87,6 +92,7 @@ namespace Microsoft::Console::Render
         ::Microsoft::WRL::ComPtr<IBoxDrawingEffect> _boxDrawingEffect;
 
         ::Microsoft::WRL::ComPtr<IDWriteFontFallback> _systemFontFallback;
+        mutable ::Microsoft::WRL::ComPtr<IDWriteFontCollection1> _nearbyCollection;

```

Comment:
> `mutable`

ew, but I get it. Almost feels like this should be static too, but idgaf

Reply:
its non static because it's tied to the factory which is a construction parameter and could theoretically change per instance so we're gonna close with no f given